CHANGELOG.md

```diff
@@ -33,6 +33,11 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 * Resolved an issue that prevented the FM-Index from being constructed correctly for inputs larger than 4 GiB
   ([\#2756](https://github.com/seqan/seqan3/pull/2756)).
 
+#### I/O
+ * **Breaking change**: Changed default of `output_options::fasta_blank_before_id` to `false`
+    ([\#2769](https://github.com/seqan/seqan3/pull/2769)).
+
+
```

Comment:
This'll need to go to `3.1.1`, and be documented via `## API changes` `#### I/O`, see `3.0.3` for example

Reply:
I think it is okay to add this fix to release 3.1.

@SGSSGene I'll change the base branch to release, think you will need to rebase to get a clean branch again :)

Reply:
It isn't really a breaking change, because in our eco-system it will accept any older written fasta file.

Reply:
> It isn't really a breaking change, because in our eco-system it will accept any older written fasta file.

Yup, but technically it changes the output